fast_pred/fastlightfm.cpp

```diff
@@ -231,3 +189,10 @@ bool FastLightFM::load(std::string dir) {
   item_features = loadCSRMatrix(cnpy::npz_load(dir + "/item-features.npz"));
   return true;
 }
+
+bool FastLightFM::is_initialized() {
```

Comment:
Is this only needed for test purpose?

Reply:
Yeah just for testing. Just wanted to stub out a test to make sure the class could be found and created really. Because I have not got the test data to pass into the constructor yet.